public/tracker.json

```diff
@@ -0,0 +1 @@
+{"20231211":34}
```

Comment:
Can we put this file in git ignore or we use database field for tracking this.

Reply:
Ignored and now using db.

Reply:
This will be migrate when updating by running: _bash bin/update_metadata.sh_